src/watertap_contrib/seto/solar_models/surrogate/trough/trough_surrogate.py

```diff
@@ -53,16 +56,36 @@ class TroughSurrogateData(SolarEnergyBaseData):
     """
 
     CONFIG = SolarEnergyBaseData.CONFIG()
+    CONFIG.declare("heat_load_range", ConfigValue(
+        domain=ListOf(float),
+        default=[10, 100],
+        description="Range of the Heat Load input",
+        doc="""Heat load range [min, max]. Options are: [10, 100] or [100, 500]"""
+    ))
+
+    surrogate_scaling_dict = {
+        (10, 100): 1e-8,
+        (100, 500): 1e-9
```

Comment:
I ended up using (10,200) and had to update this dict to run the training model and trough surrogate. I wonder if this could be updated to be more flexible?

Reply:
I agree. Since we are building in functionality to create surrogates, I wonder if we can have a default set for heat_load and storage (e.g., corresponding to the values hard-coded here in the tuples), but allow a user to choose their own bounds for the surrogate which would subsequently replace the values here.

Reply:
If we did that though, it'd probably make the workflow a lot riskier for new users who don't know what they're doing, so we might want appropriate safeguards in place as well as warnings for the user to be aware of, if we go this route.

Reply:
I agree this current proposed way of describing multiple surrogate models is clunky, but what is the intended workflow? How many surrogate models do we need for a single technology? I guess it would help to understand first what is the magnitude of any benefits to breaking up the design space into multiple models.

Regardless, I can try to put the scaling factor into the surrogate JSON directly so the data is included in the model info, rather than being separated like it is now, and maybe add some logic for the trough to choose the most appropriate surrogate model from the available ones given the input bounds.

Reply:
I am ok with this implementation for now. Really, only advanced users (i.e., us) will be using these surrogates for the time being. As we get better at building/using surrogates, we can probably implement some of this functionality. But for now, having to edit the bounds manually for a specific use case seems ok.